### DISCUSSION TITLE
<Your question or feedback about the GitHub Code Search and Code View products>

### DISCUSSION THREAD

**Discussion by Dhananjayanpariyarath** [community/community#131374] [Programming Help]
### Select Topic Area

Question

### Body

i am working on a project that using usart communication with full duplex mode , and i want to send and receive about 10 sensor data's both the side the issue that i am facing is that after a particular no of times successive communication one of the arduino not getting any receiving data or the that getting corrupted i am used serial write serial read like functions so i am unable to check the status of serial buffer registers it will be very helpful when you suggest a sample code for full duplex communication between two arduino's with out ending

Thank you

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Dhananjayanpariyarath ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
